### DISCUSSION TITLE
ZPL.Viewer Image Dark

### DISCUSSION THREAD

**Discussion by Tromed** [BinaryKits/ZPLUtility#298] [Q&A] [answered]
Hi,

I Was testing the ZPL.Viewer and when I save the generated Image it is dark. Is this intended?

Ty

**Comment by masternan** [comment, marked as answer]
set OpaqueBackground=false  in DrawerOptions ,such as following code.

      

>   private static DrawerOptions BuildDrawOption(string fontName = "NotoSansSC-SemiBold")
        {
            var fontFile = Path.Combine(AppContext.BaseDirectory, "fonts", $"{fontName}.ttf");
            var drawOptions = new DrawerOptions
            {
                FontLoader = fontName => { return SKTypeface.FromFile(fontFile); },
                OpaqueBackground = false,
                PdfOutput = false,
                RenderFormat = SKEncodedImageFormat.Png,
                RenderQuality = 100,
                Antialias = true,
                ReplaceDashWithEnDash = false
            };
            return drawOptions;
        }
